source/content/apps/velja.md

```diff
@@ -373,19 +373,13 @@ It means [“to choose”](https://en.wiktionary.org/wiki/velja) in [Old Norse](
 Velja benefits:
 
 - Free
-- It can [open links to certain services](https://twitter.com/sindresorhus/status/1519020970027401216) in their desktop app (Zoom, Microsoft Teams, Figma, etc.)
 - It can open Google Meet links in Chrome without any manual setup
-- Browser profile support for Chrome, Chromium, Edge, Vivaldi, and all their beta, dev, etc, editions (Choosy only supports Chrome stable)
 - Removes tracking parameters on clicked and copied links
-- You can open copied links from the menu bar menu
```

Comment:
I don't see any menu item for this after copying a link.

Reply:
There isn't a separate menu item, but if you click on a browser in the menu bar menu and there's a link on the clipboard, Choosy will open the link. Maybe not the most discoverable UX, but it's been a Choosy for about 14 years (since [version 0.9.2](https://choosy.app/releases/0.9.2))